deployment/README.md

```diff
@@ -82,4 +82,4 @@ docker run --rm --cap-drop ALL --cap-add=SYS_ADMIN \
 -e XPUM_REST_NO_TLS=1 \
 -e XPUM_METRICS=0-37 \
 ${xpum_image}
-```
\ No newline at end of file
+```

```

Comment:
this is unintentional

Reply:
Apparently github gui automatically adds a newline if it is missing. Doesn't matter, I suppose.